source/speech/speech.py

```diff
@@ -1854,7 +1857,7 @@ def getControlFieldSpeech(  # noqa: C901
 	hasDetailsSequence = getPropertiesSpeech(reason=reason, hasDetails=hasDetails)
 	placeholderSequence = getPropertiesSpeech(reason=reason, placeholder=placeholderValue)
 	nameSequence = getPropertiesSpeech(reason=reason, name=name)
-	valueSequence = getPropertiesSpeech(reason=reason, value=value)
+	valueSequence = getPropertiesSpeech(reason=reason, value=value, _role=role)

```

Comment:
Why was this change needed?

Reply:
Role needs to be considered in order to know that value must not be spoken. Passing in `role=role` would result in the role text being added to the output value Sequence. There is already support in this function for underscore prefixed role `_role` to mean consider it, but don't add to output.